airflow/providers/amazon/aws/example_dags/example_google_api_to_s3_transfer_advanced.py

```diff
@@ -53,12 +53,13 @@
 
 
 # [START howto_operator_google_api_to_s3_transfer_advanced_task_1_2]
-def _check_and_transform_video_ids(xcom_key, task_ids, task_instance, **kwargs):
-    video_ids_response = task_instance.xcom_pull(task_ids=task_ids, key=xcom_key)
+def _check_and_transform_video_ids(xcom_key, task_output):
+    video_ids_response = task_output[xcom_key]
     video_ids = [item['id']['videoId'] for item in video_ids_response['items']]
 
     if video_ids:
-        task_instance.xcom_push(key='video_ids', value={'id': ','.join(video_ids)})
+        context = get_current_context()
+        context["task_instance"].xcom_push(key='video_ids', value={'id': ','.join(video_ids)})
```

Comment:
Have you tested this change?

Reply:
You're right, this was a miss! I'll update accordingly.